spec/models/chargeback_rate_detail_currency_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe ChargebackRateDetailCurrency do
+  it "has a valid factory" do
+    FactoryGirl.create(:chargeback_rate_detail_currency_EUR).should be_valid
```

Comment:
We are bundling `shoulda-matchers` - I wonder if these tests could be updated to leverage those instead?